components/SearchResult.tsx

```diff
@@ -1,24 +1,29 @@
-import { useQuery } from "@tanstack/react-query";
+import { useInfiniteQuery } from "@tanstack/react-query";
 import Result from "./Result";
 import { Album, SearchResponseSchema } from "@/schemas/collections.schemas";
 import ResultLoading from "./ResultLoading";
 
 export default function SearchResult({ searchValue }: { searchValue: string }) {
-  const { data, isError, isLoading } = useQuery({
+  const { data, error, isFetching, fetchNextPage } = useInfiniteQuery({
     queryKey: ["search", searchValue],
-    queryFn: async () => {
+    queryFn: async ({ pageParam }) => {
       const params = new URLSearchParams({
-        search: searchValue,
+        search: searchValue ? searchValue : "flume",
+        page: String(pageParam),
       });
+
       const res = await fetch("/api/search?" + params);
       const data = await res.json();
       const parsedData = SearchResponseSchema.parse(data);
-
       return parsedData;
     },
+    initialPageParam: 1,
```

Comment:
I'd double check and make sure Dicog's page pagination doesn't start on 0

Reply:
I already did, it starts on 1